routing/routing_quality/routing_quality_tests/ferry_tests.cpp

```diff
@@ -15,18 +15,119 @@ UNIT_TEST(RoutingQuality_FinlandBridgeInsteadOfFerry)
 // TODO: This test doesn't pass because routing::RouteWeight::operator<
 // prefer roads with less number of barriers. It will be more useful to consider
 // barriers only with access=no/private/etc tag.
-//UNIT_TEST(RoutingQuality_RussiaToCrimeaFerry)
+//UNIT_TEST(Ferry_RoutingQuality_RussiaToCrimeaFerry)
 //{
 //  // From Russia to Crimea
 //  TEST(CheckCarRoute({45.34123, 36.67679} /* start */, {45.36479, 36.62194} /* finish */,
 //                     {{{45.3532, 36.64912}}} /* reference track */),
 //       ());
 //}
 
-UNIT_TEST(RoutingQuality_RussiaFromCrimeaFerry)
+UNIT_TEST(Ferry_RoutingQuality_RussiaFromCrimeaFerry)
 {
   TEST(CheckCarRoute({45.36479, 36.62194} /* start */, {45.34123, 36.67679} /* finish */,
                      {{{45.3532, 36.64912}}} /* reference track */),
        ());
 }
+
+// For tests Ferry_RoutingQuality_1 - Ferry_RoutingQuality_15
+// Look to: https://confluence.mail.ru/display/MAPSME/Ferries for more details.
```

Comment:
`// look at: https://confluence.mail.ru/display/MAPSME/Ferries for more details.`